.gitignore

```diff
@@ -4,6 +4,7 @@ npm-debug.log
 coverage
 test-results/
 playwright-report/
+public/custom
```

Comment:
@joshreisner  I have some demo app test pages locally, they're looking at junk google sheets on my personal account. Wanted a local directory I can save work to.